src/components/ui/Tiles/Tile.tsx

```diff
@@ -1,22 +1,22 @@
 import React, { forwardRef } from 'react'
 import type { HTMLAttributes } from 'react'
 
-export interface TileProps extends HTMLAttributes<HTMLDivElement> {
+export interface TileProps extends HTMLAttributes<HTMLLIElement> {
```

Comment:
At first I thought it was a “typo" (LI uppercase) hahaha, but it's really like [that](https://developer.mozilla.org/en-US/docs/Web/API/HTMLLIElement). 🙈